src/Connection.hpp

```diff
@@ -42,8 +42,6 @@ class Connection
   , public Serializable
 {
 
-  Q_OBJECT
```

Comment:
This is wrong. By removing the Q_OBJECT you hide this class from qt meta object generator